Sources/DashX/DashXClient.swift

```diff
@@ -435,4 +436,120 @@ public class DashXClient {
             }
         }
     }
+    
+    public func uploadExternalAsset(
+        filePath: URL,
+        externalColumnId: String,
+        successCallback: @escaping SuccessCallback,
+        failureCallback: @escaping FailureCallback
+    ) {
+        self.prepareExternalAsset(externalColumnId: externalColumnId) { response in
+            
+            if let jsonDictionary = response.jsonValue as? [String: Any],
+               let data = jsonDictionary["data"] as? [String: Any],
+               let uploadData = data["upload"] as? [String: Any],
+               let urlString = uploadData["url"] as? String,
+               let url = URL(string: urlString),
+               let assetId = jsonDictionary["id"] as? String {
+                
+                var uploadAssetRequest = URLRequest(url: url)
+                uploadAssetRequest.httpMethod = "PUT"
+                uploadAssetRequest.timeoutInterval = 2
```

Comment:
Is this timing out after 2 seconds? I think this should not be there.

Reply:
Yeah, will remove this